src/simplnx/Utilities/Meshing/TriangleUtilities.hpp

```diff
@@ -0,0 +1,101 @@
+#pragma once
+
+#include "simplnx/Common/Range.hpp"
+#include "simplnx/Common/Result.hpp"
+#include "simplnx/DataStructure/Geometry/IGeometry.hpp"
+#include "simplnx/DataStructure/Geometry/INodeGeometry2D.hpp"
+#include "simplnx/Filter/IFilter.hpp"
+
+namespace nx::core::MeshingUtilities
+{
+namespace detail
+{
+inline static constexpr usize k_00 = 0;
+inline static constexpr usize k_01 = 1;
+inline static constexpr usize k_02 = 2;
+inline static constexpr usize k_10 = 3;
+inline static constexpr usize k_11 = 4;
+inline static constexpr usize k_12 = 5;
+inline static constexpr usize k_20 = 6;
+inline static constexpr usize k_21 = 7;
+inline static constexpr usize k_22 = 8;
+
+SIMPLNX_EXPORT INodeGeometry2D::SharedVertexList::value_type FindTetrahedronVolume(const std::array<usize, 3>& vertIndices, const INodeGeometry2D::SharedVertexList::store_type& vertices);
```

Comment:
in GeometryMath.hpp there is another function with the same name. Can we document each one and probably rename one of them to that the difference is obvious.